Code/GraphMol/MonomerInfo.cpp

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (C) 2021 Greg Landrum and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <iostream>
+#include "MonomerInfo.h"
+
+using namespace RDKit;
+
+//! allows AtomPDBResidueInfo objects to be dumped to streams
+std::ostream &operator<<(std::ostream &target, const AtomPDBResidueInfo &apri) {
```

Comment:
Is there a reason not to use PDB style output?  I find it more "comfortable" due to having read too many pdb files... :)

Reply:
the only reason is that I haven't looked at that many pdb files. :-)
how would you like that formatted? (and remember that we probably shouldn't output coordinates here)

Reply:
The standard pdb format would be (I can get you formatting) but this doesn't seem like it matches with LYS-1 so it may be too limiting....

```
NE2 HIS A   1
```

Given this, I think we. can keep it and perhaps change it later if we find a better format.